packages/Webkul/Admin/src/Resources/views/customers/general.blade.php

```diff
@@ -102,4 +102,4 @@
                 </div>
             </div>
```

Comment:
Add 'bagisto.admin.customer.edit.form.after' here

Reply:
added after this div